meinberlin/assets/scss/components/_lr_bar.scss

```diff
@@ -7,8 +7,11 @@
 
 .lr-bar__left {
     display: inline-block;
+    margin: 0;
```

Comment:
This seems to solve a specific problem (headings with margin-top), but I am not sure if it's generally applicable, since it also only works if you put that class directly on the heading. I'd leave it in for now but we should be aware of that.